client/my-sites/plugins/plugin-site/plugin-site.jsx

```diff
@@ -21,4 +24,24 @@ const PluginSite = ( props ) => {
 	return <PluginSiteJetpack { ...props } />;
 };
 
-export default PluginSite;
+export default connect(
+	( state, ownProps ) => {
+		const pluginSlug = get( ownProps, 'plugin.slug' );
+		const siteId = get( ownProps, 'site.ID' );
+
+		if ( includes( [ 'jetpack', 'vaultpress' ], pluginSlug ) && isSiteAutomatedTransfer( state, siteId ) ) {
+			return {
+				...ownProps,
+				...{
+					allowedActions: {
+						activation: false,
+						autoupdate: false,
+						remove: false,
+					}
+				},
+				isAutoManaged: true,
+			};
+		}
+		return ownProps;
```

Comment:
Oh just noticed that we were doing something odd with ownProps. What are we trying to override that the default [mergeProps](https://github.com/reactjs/react-redux/blob/master/docs/api.md#connectmapstatetoprops-mapdispatchtoprops-mergeprops-options) doesn't handle? 

For reference mergeProps has a default implementation of that is passed to the component:
```
function mergeProps( stateProps, dispatchProps, ownProps ) {
    Object.assign({}, ownProps, stateProps, dispatchProps);
}
```

Which properties are we trying to override?